plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/maven2/metadata/MetadataRebuilder.java

```diff
@@ -59,10 +58,9 @@
 import com.orientechnologies.orient.core.metadata.schema.OType;
 import com.orientechnologies.orient.core.record.impl.ODocument;
 import com.orientechnologies.orient.core.sql.query.OSQLAsynchQuery;
-import com.orientechnologies.orient.core.tx.OTransaction.TXTYPE;
-import org.w3c.dom.Document;
-import org.w3c.dom.NodeList;
-import org.xml.sax.SAXException;
+import org.codehaus.plexus.util.xml.Xpp3Dom;
```

Comment:
FTR Maven impl may still want/need to use various plexus-shits since that is what Maven itself uses.  

Everything else however, should avoid it like the plague.